docs/src/main/asciidoc/security-authorize-web-endpoints-reference.adoc

```diff
@@ -182,6 +182,17 @@ You can also create global `HttpSecurityPolicy` invoked on every request.
 Just do not implement the `io.quarkus.vertx.http.runtime.security.HttpSecurityPolicy.name` method and leave the policy nameless.
 ====
 
+[[policy-active-cdi-request-context]]
+=== Inject `@RequestScoped` beans into HttpSecurityPolicy
```

Comment:
```suggestion
=== Inject `@RequestScoped` beans into `HttpSecurityPolicy`
```

Reply:
@michalvavrik Would you like to apply this one ?

Reply:
sorry, I read this one and messed up somehow. fixed